lib/rucio/core/rule.py

```diff
@@ -923,26 +923,30 @@ def list_associated_rules_for_file(scope, name, session=None):
 
 
 @transactional_session
-def delete_rule(rule_id, purge_replicas=None, soft=False, delete_parent=False, nowait=False, session=None):
+def delete_rule(rule_id, purge_replicas=None, soft=False, delete_parent=False, nowait=False, session=None,
+                ignore_rule_lock=False):
     """
     Delete a replication rule.
 
-    :param rule_id:         The rule to delete.
-    :param purge_replicas:  Purge the replicas immediately.
-    :param soft:            Only perform a soft deletion.
-    :param delete_parent:   Delete rules even if they have a child_rule_id set.
-    :param nowait:          Nowait parameter for the FOR UPDATE statement.
-    :param session:         The database session in use.
-    :raises:                RuleNotFound if no Rule can be found.
-    :raises:                UnsupportedOperation if the Rule is locked.
+    :param rule_id:           The rule to delete.
+    :param purge_replicas:    Purge the replicas immediately.
+    :param soft:              Only perform a soft deletion.
+    :param delete_parent:     Delete rules even if they have a child_rule_id set.
+    :param nowait:            Nowait parameter for the FOR UPDATE statement.
+    :param session:           The database session in use.
+    :param ignore_rule_lock:  Ignore any locks on the rule
+    :raises:                  RuleNotFound if no Rule can be found.
+    :raises:                  UnsupportedOperation if the Rule is locked.
     """
 
     with record_timer_block('rule.delete_rule'):
         try:
-            rule = session.query(models.ReplicationRule).filter(models.ReplicationRule.id == rule_id).with_for_update(nowait=nowait).one()
+            rule = (session.query(models.ReplicationRule)
```

Comment:
It's only cosmetics, but can you please remove the enclosing parenthesis? Just for symmetry with the rest of the code. (The line-breaks are fine)